IPython/core/hooks.py

```diff
@@ -90,6 +91,14 @@ def fix_error_editor(self,filename,linenum,column,msg):
 
     Call ip.set_hook('fix_error_editor',youfunc) to use your own function,
     """
+
+    warnings.warn("""
+`fix_error_editor` is pending deprecation as of IPython 5.0 and will be removed
+in future versions. It appear to be used only for automatically fixing syntax
+error that has been broken for a few years and has thus been removed. If you
+append to use this function and still need it please make your voice heard.
```

Comment:
append -> happened

Also, let's be more specific about 'make your voice heard', e.g. point them to the issue tracker.